fastlane/metadata/steam/czech/description.txt

```diff
@@ -0,0 +1,61 @@
+Vytvoř propracovaný výrobní řetězec dopravníků, abys naládoval munici do věží, vyrobil materiály pro stavění a ubránil své stavby před vlnami nepřátel. Hraj s Tvými přáteli koperačně napříč platformami, nebo je vyzvi v týmové hře hráčů proti sobě.
+
+[img]{STEAM_APP_IMAGE}/extras/ezgif-4-0e70c282f775.gif[/img]
+
+[h2]Hra[/h2]
+
+[list]
+[*] Vytvoř vrty a dopravníky pro přesun zdrojů do Tvého jádra. 
+[*] Použij výrobní bloky pro vytvoření pokročilých materiálů.
+[*] Postav drony pro automatické těžení zdrojů, pomoc se stavěním a vybudování a ochraně své základny.
+[*] Rozváděj kapaliny a bojuj s požárem.
+[*] Posilni výrobu dodatečným chlazením a promaž obranné i výrovní bloky.
+[/list]
+
+[h2]Kampaň[/h2]
+
+[list]
+[*] Projdi skrz 12 znovuhratelných předpřipravených zón s náhodným místem startu
```

Comment:
V seznamu výše je za každou větou tečka, a tady (a v seznamech níže) není. Šlo by to prosím sjednotit?

Reply:
v originále je to takhle, jednou s a jednou bez, ale sjednotil jsem to na tečku a dva vykřičníky :))